linuxclientsetup/scripts/client-config

```diff
@@ -293,6 +293,16 @@ if [[ -L /usr/lib/firefox/dictionaries ]]; then
 	ln -s /usr/share/hunspell/"$machineLang".aff /usr/lib/firefox/dictionaries/"$machineLang".aff
 fi
 
+#Lockdown preferences
+if [[ ! -L /usr/lib/firefox/mozilla.cfg ]]; then
+	cat << EOF > /var/lib/karoshi/firefox/mozilla.cfg
+//
+lockPref('xpinstall.enabled', false);
+lockPref('browser.startup.homepage', '$PROXYSERVER.$DNSSUFFIX');
+EOF
+	ln -s /var/lib/karoshi/firefox/mozilla.cfg /usr/lib/firefox/mozilla.cfg
```

Comment:
The link creation is unnecessary - it already exists in the `configuration` directory